pkg/host/syscalls_linux.go

```diff
@@ -223,13 +223,13 @@ func isSupportedSyzkall(sandbox string, c *prog.Syscall) (bool, string) {
 	case "syz_execute_func":
 		return true, ""
 	case "syz_io_uring_submit":
-		// TODO: check if io_uring syscall is present
-		return true, ""
+		fallthrough
```

Comment:
Since we don't have any code in these cases, it will be shorter/cleaner to combine them into a single case:
```
case "syz_io_uring_submit", "syz_io_uring_complete", "syz_memcpy_off":
```

Reply:
[Fixed](https://github.com/google/syzkaller/pull/1926/commits/b9950dc9fb2fd181607037210a85b221ec09d335)